# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/compiler/preprocessor/DirectiveHandlerBase.h
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/compiler/preprocessor/DirectiveHandlerBase.h
--- a/gfx/angle/checkout/src/compiler/preprocessor/DirectiveHandlerBase.h	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/compiler/preprocessor/DirectiveHandlerBase.h	Sat Sep 07 13:38:36 2019 +0000
@@ -8,6 +8,7 @@
 #define COMPILER_PREPROCESSOR_DIRECTIVEHANDLERBASE_H_
 
 #include <string>
+#include "GLSLANG/ShaderLang.h"
 
 namespace angle
 {
@@ -38,7 +39,7 @@
                                  const std::string &name,
                                  const std::string &behavior) = 0;
 
-    virtual void handleVersion(const SourceLocation &loc, int version) = 0;
+    virtual void handleVersion(const SourceLocation &loc, int version, ShShaderSpec spec) = 0;
 };
 
 }  // namespace pp